css/styles.css

```diff
@@ -92,3 +92,8 @@ textarea {
 .new-button {
   float: none;
 }
+
+.far.fa-edit {
```

Comment:
You should not apply styling directly to classes from libraries like this. If another developer on your team attempts to use the library in their code and they need an edit icon, it will have unexpected styling attached to it. Please create a custom class for this styling that you can apply to the icon elements.